test/parallel/test-fs-mkdtemp.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+common.refreshTmpDir();
+
+const tmpFolder = fs.mkdtempSync(path.join(common.tmpDir, 'foo.'));
+
+assert(path.basename(tmpFolder).length === 'foo.XXXXXX'.length);
+assert(common.fileExists(tmpFolder));
```

Comment:
Ok. The test already runs fine on my computer, I'll try to find time to push this.